pod/perldiag.pod

```diff
@@ -6829,13 +6829,15 @@ discovered.  See L<perlre>.
 
 =item Unknown Unicode option letter '%c'
 
-(F) You specified an unknown Unicode option.  See L<perlrun> documentation
-of the C<-C> switch for the list of known options.
+(F) You specified an unknown Unicode option.  See
+L<perlrun|perlrun/-C [numberE<sol>list]> documentation of the C<-C> switch
```

Comment:
> "Unknown escape: E<sol> in (F) You specified an unknown Unicode option...

This pod file is used by diagnostics.pm to generate the list of strings -- so its syntax is a bit finicky. It may be that you cannot nest pod markup inside other markup here?

Reply:
The problem is diagnostics.pm does its own E<> resolution, and the table it uses doesn't include `sol` or `verbar`, which seem to be required in L<> according to perlpod.